src/xref2/ref_tools.ml

```diff
@@ -23,30 +23,40 @@ type type_lookup_result =
   | `C of class_lookup_result
   | `CT of class_type_lookup_result ]
 
-type value_lookup_result = Resolved.Value.t
```

Comment:
If we don't need these let's get rid of them rather than just commenting them out.